torch_xla/utils/gcsfs.py

```diff
@@ -220,6 +243,7 @@ def write(path, content):
 
 def generic_write(output_string, output_path):
   """Write a string/bytes or file into a GCS blob or local disk, depending
+
```

Comment:
nit: newline not needed.

Reply:
Actually, it is.
The comment is wrong. Head comment must be a one liner, so google-pyformat tried to fix that.

Reply:
Fixed this.